### ISSUE TITLE
Delta as pager is much slower after 0.41.0

### ISSUE THREAD

**Issue by ptim** [jesseduffield/lazygit#3473]
**Describe the bug**

Amazing release, thanks! 

I have [delta configured](https://github.com/jesseduffield/lazygit/blob/master/docs/Custom_Pagers.md) as my pager, and it's now very slow - it takes nearly a second for the diff pane to update with each new diff.

Workaround: Reverting to the built in pager restores performance

**To Reproduce**

- configure delta as below
- navigate between changed files in the File tab

```
# ~/Library/Application\ Support/lazygit/config.yml

git:
  paging:
    pager: delta --paging=never
    colorArg: always
```

```
# ~/.gitconfig

[delta]
  navigate = true
  line-numbers = true
  features = zebra-dark
```

**Expected behavior**

Snappy diffs in delta, like before :)

**Version info:**

commit=, build date=, build source=homebrew, version=0.41.0, os=darwin, arch=arm64, git version=2.39.3 (Apple Git-145)

git version 2.39.3 (Apple Git-145)

**Additional context**

I ran `lazygit --debug` in one terminal panel and `lazygit --logs` in another, 
but didn't see any logs when changing the focussed file in the Files pane.

**Comment by mark2185** [comment]
It was fixed in #3420 so you can either build from `master` or add `--dark` or `--light` to your pager field in the config.

**Comment by ptim** [comment]
Love an easy fix, many thanks - adding `--dark` works a treat!

Ah, I see it was reported earlier here: https://github.com/jesseduffield/lazygit/issues/3405
